templates/components/itilobject/timeline/pending_reasons_messages.html.twig

```diff
@@ -39,12 +39,31 @@
    {% set pending_item = pending_item ?? call(entry['type'] ~ "::getById", [entry_i['id']]) %}
    {% set pending_reason_item = pending_item ? call("PendingReason_Item::getForItem", [pending_item]) : false %}
    {% set pending_reason = pending_reason_item ? call("PendingReason::getById", [pending_reason_item.fields['pendingreasons_id']]) : false %}
+   {% set is_latest_pending = call("PendingReason_Item::isLastPendingForItem", [
```

Comment:
When the ticket is no longer pending, it should be ‘done’ everywhere, shouldn't it?

![image](https://github.com/user-attachments/assets/b7acaaf4-0c22-41d7-b8ff-7b77abe9bb7b)